### ISSUE TITLE
Captchas are a show stopper

### ISSUE THREAD

**Issue by hellomiakoda** [EionRobb/purple-discord#359]
Maybe something is wonky with my discord account, maybe I'm a total moron despite 135 IQ and using various linux distros since '06, but I can't get through the instructions to get passed the captcha in any way that results in this plug in working. That token never appears

**Comment by EionRobb** [comment]
So the captcha thing can be a nuisance that's hiding a different problem, the login could also be asking for a recaptcha login, on top of the captcha login, but unfortunately recaptcha relies on javascript, which Pidgin doesn't support

You could try https://github.com/EionRobb/purple-discord#bitlbee-and-spectrum2-users as a workaround - but to edit accounts.xml you'll need to make sure Pidgin is closed

**Comment by chainria** [comment]
Yeah... could we maybe please have the setting for the token shown in advanced config? That would help a lot, especially now.

**Comment by EionRobb** [comment]
@chainria yup, fair - it's specifically hidden for Pidgin in https://github.com/EionRobb/purple-discord/blob/master/libdiscord.c#L7108-L7112 since it looks pretty ugly/scary having it visible, but might as well

Or would it be better if there was a popup that asked to copy-paste it in from the browser?

**Comment by chainria** [comment]
I would LOVE the popup! This would surely be a lot better than anything else. Nice idea!

**Comment by guglovich** [comment]
Pidgin deletes the token each time it starts up. Hopefully this plugin will be simplified at some point.
